src/chai/RajaExecutionSpacePlugin.cpp

```diff
@@ -76,6 +76,21 @@ RajaExecutionSpacePlugin::postLaunch(RAJA::util::PluginContext)
 
 }
 
+// this is needed to link a dynamic lib as RAJA does not provide an exported definition of this symbol.
```

Comment:
Should we do that?

Reply:
I'm honestly not sure what the right way to handle all of this is. I'm also not sure if this would be an issue if we weren't doing the slightly weird thing of building RAJA static and chai dynamic in our build system (don't ask).

Reply:
sleeping on this - yeah it'd be good if RAJA had the option to export this symbol. But I think we can follow up with a separate PR in RAJA and do it in a way that will clean this up but not break this change to CHAI.

Reply:
@robinson96 or @davidbeckingsale can one of you put up a RAJA GitHub issue for this?  Thanks.